### PR TITLE
Allow passing arguments like verify=False to requests

### PR DESCRIPTION
**Pull request by jpmckinney**
e.g. https://opendata.riik.ee/ has a bad cert.

### CODE REVIEW

File: ckanapi/remoteckan.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def __init__(self, address, apikey=None, user_agent=None, get_only=False):
             self.parallel_limit = PARALLEL_LIMIT
 
     def call_action(self, action, data_dict=None, context=None, apikey=None,
-            files=None):
+            files=None, **kwargs):
```
Comment (wardi) [inline ckanapi/remoteckan.py]: Change this to `, requests_kwargs=None)` and I'll merge it.

Sorry for the slow reply @jpmckinney

Comment (jpmckinney) [inline ckanapi/remoteckan.py]: Had to be `, requests_kwargs={}` otherwise the eventual `**` complains.

Comment (wardi) [inline ckanapi/remoteckan.py]: Just do a `requests_kwargs = requests_kwargs or {}` in the body. Mutable default arguments freak me out

Comment (jpmckinney) [inline ckanapi/remoteckan.py]: Good catch. Done.

**Pull request merged**